### ISSUE TITLE
Project Manager Feature Icon buttons not accessible

### ISSUE THREAD

**Issue by finchnSNPs** [OSC/ondemand#5173]
We've been assessing our deployment of Open OnDemand at University of Washington. We have hidden the Job Composer feature since it is deprecated and continues to have low accessibility scores. In our assessment of the, "Create New Project" tool, we found many issues - every icon button. 

<img width="2048" height="999" alt="Image" src="https://github.com/user-attachments/assets/cb116a63-92b2-40e7-ac56-54c5efc5554d" />

First, the icon picker is not navigable via keyboard. When I have the picker selected, I cannot use the Enter/Return button to select an icon. Additionally, from my testing, even though a "cog" is shown as if it is the default option, you cannot save the project without selecting an icon. I think this is because the icon picker is an li html element. 

Typing in the icon does work, but these icons don't have accessible labels so being able to select one by knowing what it is called, is difficult and not screenreader compatible. 

The rule: “Ensure every ARIA button, link and menu item has an accessible name”.

Recommendation, add aria-label for all icons inside the icon picker list. Example: 
 
```
<li id="icon_ambulance" class="btn btn-outline-dark" role="button" aria-label="ambulance">
            <i class="fas fa-ambulance fa-fw"></i>
          </li>
```
 
It is a multi-pronged; however, we do like the Project Manager in principle. 

Is there a timeline for making accessible improvements to this feature? 

Thank you, 

Kristen Finch
Director of Research Computing Solutions
UWIT

**Comment by achalker** [comment]
Kristen:  We've publicly announced / committed to releasing version 4.2 of Open OnDemand in early/mid April that will be fully compliant with the digital accessibility requirements that come into full effect on April 24th for many people.  In case you aren't aware, we did issue a ACR (VPAT) with the 4.1 release that documents the known issues:  https://www.openondemand.org/accessibility

**Comment by finchnSNPs** [comment]
Thanks, Alan. Project Manager wasn't mentioned specifically, but I'm glad these improvements are on the roadmap. I will hold off on [posting additional GH issues identifying gaps](https://discourse.openondemand.org/t/w3c-2-1aa-accessability-standards/4655/5?u=finchnsnps) until version 4.2.

**Comment by johrstrom** [comment]
> I will hold off on [posting additional GH issues identifying gaps](https://discourse.openondemand.org/t/w3c-2-1aa-accessability-standards/4655/5?u=finchnsnps) until version 4.2.

No please post whatever you find. We may have fixed somethings since 4.1, but even so closing a ticket as already done is much better than never knowing or delaying the finding altogether.
